dotCMS/src/main/resources/oracle.sql

```diff
@@ -2502,7 +2502,8 @@ CREATE TABLE system_event (
 	identifier VARCHAR(36) NOT NULL,
 	event_type VARCHAR(50) NOT NULL,
 	payload NCLOB NOT NULL,
-	created NUMBER(19, 0) NOT NULL
+	created NUMBER(19, 0) NOT NULL,
+  server_id varchar2(36) NOT NULL
```

Comment:
Based on the **identifier** column I think the correct type for the **server_id** is **VARCHAR** instead of **varchar2**

Reply:
Nevermind, **varchar2** looks correct, looks like is **identifier** the one that should be a **varchar2** instead of **VARCHAR** so please ignore the comment above..

Reply:
ok, I just copied the definition from the server table!